scipy/optimize/_lsq/least_squares.py

```diff
@@ -744,6 +746,7 @@ def least_squares(
     We wrap it into a function of real variables that returns real residuals
     by simply handling the real and imaginary parts as independent variables:
 
+    >>> import numpy as np
```

Comment:
I think this import statement is not needed because of the rule: "Add them a maximum of once per "Example" section."